### DISCUSSION TITLE
Convert to Issue

### DISCUSSION THREAD

**Discussion by sherryumlah** [community/community#5558] [Projects and Issues]
Converting to an issue was not rendering the assignees list of names.  I had to refresh the screen for this to display. 
 After refreshing, converting a draft item to an issue and assigning a repo and team member results in a 422 status code error in the console.

app.bundle.js:2 GET https://github.com/orgs/harvesthq/projects/beta/23/items/suggestions/assignees?memexProjectItemId=100687 422

**Comment by shiftkey** [comment]
:wave: thanks for the report! We have an idea of what's happened here and are investigating what the required fix looks like.

**Comment by mmclead** [comment]
same here.  I made a new item on my project and, when trying to convert it to an issue, am getting this Unhandled Promise Rejection:  `Unhandled Promise Rejection: ac: A content type of Issue or PullRequest is required`
